homeassistant/components/heos/config_flow.py

```diff
@@ -74,3 +81,24 @@ async def async_step_user(
             data_schema=vol.Schema({vol.Required(CONF_HOST, default=host): host_type}),
             errors=errors,
         )
+
+    async def async_step_reconfigure(
+        self, user_input: dict[str, Any] | None = None
+    ) -> ConfigFlowResult:
+        """Allow reconfiguration of entry."""
+        await self.async_set_unique_id(DOMAIN)
+
+        entry = self._get_reconfigure_entry()
+        host = entry.data[CONF_HOST]  # Get current host value
```

Comment:
```suggestion
```
as discussed, we can keep the different steps, but let's move that then down, because I think it's confusing here